be/src/olap/storage_engine.h

```diff
@@ -202,6 +203,15 @@ class StorageEngine {
 
     RowsetTypePB compaction_rowset_type() const { return _compaction_rowset_type; }
 
+    bool set_default_rowset_type_to_beta() {
```

Comment:
The name `set_default_rowset_type_to_beta` looks like we
`set the flag`, but what you mean is `is the flag set?`
I think `is_default_rowset_type_beta()` maybe better?

Reply:
yes, I change it